keyboards/boardnotbored/350mimi/350mimi.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include "quantum.h"
+
+#define KNO KC_NO
```

Comment:
Nit pick, but: 
```suggestion
#define ___ KC_NO
```

This can be easier to read/find, when there are issues.